pywren/scripts/pywrencli.py

```diff
@@ -209,10 +214,57 @@ def create_instance_profile(ctx):
     instance_profile_name = config['standalone']['instance_profile_name']
 
     iam = boto3.resource('iam')
-    iam.create_instance_profile(InstanceProfileName=instance_profile_name)
+    iamclient = boto3.client('iam')
+    response = iamclient.list_instance_profiles()
```

Comment:
Travis doesn't have permissions to do this. Maybe we should use try/except after all?

Reply:
can we add IAM permissions for travis to do this ? @Vaishaal ?

Reply:
We migrated Travis to a new account and added the necessary permissions.